private/type.scrbl

```diff
@@ -65,8 +66,8 @@ exactly the same fields.
 
 @subsection{Struct-Based Types and Struct Type Properties}
 
-All types are created using Racket @tech{structs}, and the created struct types
-can have @tech{structure type properties} attached to them. Each module
+All types are created using Racket @racket{struct}s, and the created struct types
```

Comment:
Instead of using `@racket` here, this probably ought to be `@tech{structure types}` since that links to the `structure type` term defined here: https://docs.racket-lang.org/reference/structures.html